README.md

```diff
@@ -12,6 +12,7 @@ We want examples of as many use cases in this repository as possible! Submit a P
 ### Basic FireLens examples
 * [Using the 'file' 'config-file-type'](examples/fluent-bit/config-file-type-file)
 * [Cross Account Log Forwarding](examples/fluent-bit/cross-account)
+* [Multiple Config Support](examples/fluent-bit/multi-config-support)
```

Comment:
Same comment as on your use case guide, let's make this title better and longer to advertise teh feature. 

Source multiple configs from S3 or files

Or a better title if you can think of one.